core/tests/protractor_utils/users.js

```diff
@@ -48,7 +54,6 @@ var logout = function() {
 // that this will fail if the user already has a username.
 var _completeSignup = function(username) {
   browser.get('/signup?return_url=http%3A%2F%2Flocalhost%3A9001%2F');
-  waitFor.pageToFullyLoad();
```

Comment:
This waits for the loading message to disappear. I could not find `ng-show="loadingMessage"` in the signup page so removed this. Also, we already have a waitFor.visibilityOf(inputField) below.